src/FormBuilder/resources/js/form-builder/src/common/registerBlocks.ts

```diff
@@ -1,4 +1,26 @@
-import {BlockSupports, registerBlockType} from '@wordpress/blocks';
+import {BlockSupports, registerBlockType, setUnregisteredTypeHandlerName} from '@wordpress/blocks';
+import {__experimentalGetCoreBlocks} from '@wordpress/block-library';
+
+/**
+ * Registers the missing block from WordPress core.
+ *
+ * @see https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/missing
+ *
+ * @unreleased
+ */
+const registerMissingBlock = () => {
+    //TODO: should probably replace this with a custom block
```

Comment:
I agree with a custom block in this case. Since this will most likely be an add-on that was deactivated, we can handle the user messaging in a much more productive way.